.gitignore

```diff
@@ -257,3 +257,6 @@ dist
 
 # PyCharm project files
 .idea/
+
+# Local settings
+muni_portal/settings/local.py
```

Comment:
I would rather recommend local dev settings using .env - or are you not using docker-compose?